test/e2e/server-startup-control/test.js

```diff
@@ -17,7 +17,7 @@ test('`fusion dev` proxy gracefully recovers from cached SSR errors', async () =
   async function waitForCompileToStart() {
     // Once the Fusion.js application renders SSR error pages in the app
     // we should leverage module.hot.addStatusHandler.
-    await new Promise(resolve => setTimeout(resolve, 3000));
+    await new Promise(resolve => setTimeout(resolve, 10000));
```

Comment:
Is this still necessary? Seems this is not a timeout, and  adjusting this has a slight performance impact on tests. (Hopefully  in the future we change this so we don't need hard-coded number at all)

Reply:
Yeah this was probably unnecessary. I put up a PR to undo https://github.com/fusionjs/fusion-cli/pull/757